graphene/pyutils/dataclasses.py

```diff
@@ -0,0 +1,1260 @@
+# This is a polyfill for dataclasses
```

Comment:
Why not add a dependency on the `dataclasses` lib?

Reply:
dataclasses are not available since Python 3.7

Reply:
No I mean the dataclasses library on PyPi: https://pypi.org/project/dataclasses/

Reply:
We might want to customize it a bit, so I think is better to ship it with the lib for now and remove it later :)